pymobiledevice3/services/mobile_config.py

```diff
@@ -23,23 +27,40 @@ def hello(self) -> None:
     def flush(self) -> None:
         self._send_recv({'RequestType': 'Flush'})
 
+    def escalate(self, pkcs12: bytes, password: str) -> None:
+        decrypted_p12 = load_pkcs12(pkcs12, password.encode('utf-8'))
+
+        escalate_response = self._send_recv({
+            'RequestType': 'Escalate',
+            'SupervisorCertificate': decrypted_p12.cert.certificate.public_bytes(Encoding.DER)
+        })
+        signed_challenge = PKCS7SignatureBuilder().set_data(escalate_response['Challenge']).add_signer(
+            decrypted_p12.cert.certificate, decrypted_p12.key, hashes.SHA256()).sign(Encoding.DER, [])
+        self._send_recv({'RequestType': 'EscalateResponse',
+                        'SignedRequest': signed_challenge})
+        self._send_recv({'RequestType': 'ProceedWithKeybagMigration'})
+
     def get_stored_profile(self, purpose: Purpose = Purpose.PostSetupInstallation) -> Mapping:
         return self._send_recv({'RequestType': 'GetStoredProfile', 'Purpose': purpose.value})
 
     def store_profile(self, profile_data: bytes, purpose: Purpose = Purpose.PostSetupInstallation) -> None:
-        self._send_recv({'RequestType': 'StoreProfile', 'ProfileData': profile_data, 'Purpose': purpose.value})
```

Comment:
Sorry for telling it too late, but we use 120 characters line length so no need to change these lines

Reply:
Ah, no worries. I know myself how hard it is to maintain an open source project and keep track of all the unwritten contributing rules (or writing them down for that matter). Thank you for keeping this project running!

I should have fixed all the code style issue now.